### PR TITLE
Use quick pairing buttons to seek games

### PR DESCRIPTION
**Pull request by ddugovic**
https://lishogi.org/forum/lishogi-feedback/request-implementation-of-automatic-pairing-function--

### CODE REVIEW

**Comment by WandererXII** [comment]
While I definitely see the appeal, I don't think it's sustainable with the current number of players.
We would have to somewhat tweak it, maybe making it just a preset for lobby would work? idk...

**Comment by TheYoBots** [comment]
2|0(bullet), 0|10(bullet), 5|0(blitz) are quite commonly played time controls. Maybe if you're open to replacing a few time controls you have added with these ones (especially 0|10).

File: ui/lobby/src/boot.ts [hunk 1]
```diff
@@ -1,7 +1,17 @@
 export default function boot(cfg, element) {
   cfg.pools = [
     // mirrors modules/pool/src/main/PoolList.scala
-    //{ id: "1+0", lim: 1, inc: 0, perf: "Bullet" }
+    { id: '0|5', lim: 0, inc: 0, byo: 5, perf: 'Bullet' },
+    { id: '2+2', lim: 2, inc: 2, byo: 0, perf: 'Bullet' },
+    { id: '3|5', lim: 3, inc: 0, byo: 5, perf: 'Blitz' },
+    { id: '5+3', lim: 3, inc: 2, byo: 0, perf: 'Blitz' },
+    { id: '5|10', lim: 5, inc: 0, byo: 5, perf: 'Blitz' },
+    { id: '10+10', lim: 5, inc: 3, byo: 0, perf: 'Rapid' },
```
Comment (TheYoBots) [inline ui/lobby/src/boot.ts:9]: The `lim`, `inc`, and `byo` are wrong above.

```ts
    { id: '5+3', lim: 5, inc: 3, byo: 0, perf: 'Blitz' },
    { id: '5|10', lim: 5, inc: 0, byo: 10, perf: 'Blitz' },
    { id: '10+10', lim: 10, inc: 0, byo: 10, perf: 'Rapid' },
```

**Comment by ddugovic** [comment]
Today I learned that scaling may be helped by
https://github.com/WandererXII/lishogi/blob/9175e4e2aa3387bb13087f30f4ee089f6b2060b0/modules/lobby/src/main/LobbyTrouper.scala#L45-L46

which years ago I heard about... players were afraid of using Quick Pairing, so it was made to match compatible game seeks.

**Comment by ddugovic** [comment]
Having seen this feature... you're right, this solution is designed for a different problem (having too many game seeks) and won't increase lobby activity.

**Comment by ddugovic** [comment]
Your suggestion of using presets was easier to implement than I had planned, and seems to work well:
* Anonymous players can seek Casual games and get paired with other anonymous players
* Logged in players can seek Rated games and get paired with other logged in players

**Comment by WandererXII** [comment]
Sorry for leaving this here for so long. With your implementation in mind I implemented something similar and created this issue to discuss it more: https://github.com/WandererXII/lishogi/issues/782. Thanks!

**Comment by ddugovic** [comment]
No worries at all! You're doing great maintaining this project, and things worth doing are worth doing correctly/well and can take time...

**Pull request closed without merging**